doc/backups.md

```diff
@@ -51,21 +51,21 @@ For a stateboard instance only snapshot files and instance configuration file wi
 
 To restore you can use the `restore` step. It contains three stages:
 
-* find TGZ backup or folder with backup ([details here](#find-backup));
+* find TGZ backup or directory with backup ([details here](#find-backup));
 * check for conflicting files ([details here](#check-for-conflicting-files));
 * replace current files with files from backup ([details here](#replace-current-files-with-files-from-backup)).
 
 Examples of restoring from backup, you can see [below](#using-restore-step).
 
 ### Find backup
 
-1. You can pass path of backup by `cartridge_restore_backup_path` variable.
-   Just set the value of this variable to full path of TGZ or folder backup.
+1. You can pass path to backup by `cartridge_restore_backup_path` variable.
+   Path to a TGZ archive or a directory with backup can be specified.
 2. If `cartridge_restore_backup_path` variable is omitted, the role
-   will look in the `cartridge_remote_backups_dir` folder
+   will look in the `cartridge_remote_backups_dir` directory
    (by default `/opt/tarantool/backups`) for the backup
-   whose name is the most recent alphabetically and contains the instance ID.
-3. If no file or folder is selected, an error will occur.
+   whose name contains the instance ID and is the latest alphabetically (with the biggest date).
+3. If no file or directory is selected, an error will occur.
```

Comment:
What does `selected` mean? Maybe `If not found any file or directory backup that belongs to given instance, ...`?